packages/nux/README.md

```diff
@@ -20,7 +20,7 @@ npm install @wordpress/nux --save
 
 See [the component's README][dot-tip-readme] for more information.
 
-[dot-tip-readme]: https://github.com/WordPress/gutenberg/tree/master/nux/components/dot-tip/README.md
+[dot-tip-readme]: https://github.com/WordPress/gutenberg/tree/master/packages/nux/components/dot-tip/README.md
```

Comment:
components folder is under `src`

Correct URL:
https://github.com/WordPress/gutenberg/blob/master/packages/nux/src/components/dot-tip/README.md